### ISSUE TITLE
Suggestion: Merchant Stall

### ISSUE THREAD

**Issue by BlkHellfire** [TangoTek/TekTopia-Community#282]
It would be nice if there was a stall that the merchant would stay at throughout the day. It can be quite tedious searching for the merchant, seeing what his trades are, and  than finding him again after you have gotten the items to trade with. My suggestion is to have an option to buy a merchant stall token where the merchant will stay at during the day.

**Comment by JohnyDL** [comment]
I offered a different way to search for villagers, see #190 and I know somewhere Tango mentioned that the Merchant might be getting something special... found it #133 the issue is closed for that bug but yeah there might be something coming to help

**Comment by Konyv** [comment]
That would be great.

**Comment by ravenclawnc12** [comment]
Maybe the merchant does not stay all day, but will go back to the stall multiple times each day.... Also an auto trade option so a chest in said stall for when the merchant has the corresponding item, it just sells.

**Comment by JohnyDL** [comment]
Yup Ravenclawnc12 I mentioned that in #133 go look at Tango's response ^_^
